changelogs/current.yaml

```diff
@@ -125,6 +125,9 @@ bug_fixes:
 - area: health_checker
   change: |
     prevent writing pending data for health checkers by introducing ``ConnectionCloseType::Abort`` to avoid cascading handshake overhead from health checker's requests on timeout. This fix is related to `issue \#23718 <https://github.com/envoyproxy/envoy/issues/23718>`_.
+- area: tcp_proxy
+  change: |
```

Comment:
Let's change that first sentence to

When tunneling HTTP over TCP, mark the upstream connection as done reading when upstream trailers are read.